### PR TITLE
Update MMH uk_ua.json

### PR DESCRIPTION
**Pull request by Den4enko**

### CODE REVIEW

**Comment by RozeFound** [comment]
`modmenu.summaryTranslation.modelfix` string is updated in [en_us.json](https://github.com/Fabulously-Optimized/fabulously-optimized/blob/main/Resourcepacks/Mod%20Menu%20Helper/assets/fo/lang/en_us.json) file, don't forget to update it too.

**Comment by Den4enko** [comment]
> `modmenu.summaryTranslation.modelfix` string is updated in [en_us.json](https://github.com/Fabulously-Optimized/fabulously-optimized/blob/main/Resourcepacks/Mod%20Menu%20Helper/assets/fo/lang/en_us.json) file, don't forget to update it too.

Thanks! I have just fixed it!

File: Resourcepacks/Mod Menu Helper/assets/fo/lang/uk_ua.json [hunk 1]
```diff
@@ -40,7 +41,8 @@
   "modmenu.summaryTranslation.mixin-conflict-helper": "Сповіщення про конфлікт модів\n§8✎ (без налаштувань)",
   "modmenu.summaryTranslation.mixintrace": "Допомагає розробникам знайти причину вилетів\n§8✎ (без налаштувань)",
   "modmenu.summaryTranslation.modmenu": "Це меню модів\n§9✎§8 Зміна росташування, фільтр, сортирування, ...",
-  "modmenu.summaryTranslation.modelfix": "Виправлення прозорих ліній в предметах\n§8✎ (без налаштувань)",
+  "modmenu.summaryTranslation.moreculling": "Припинення рендеру схованних сторін блоків\n§9✎§8 Перемикання блоків, рамки, ...",
+  "modmenu.summaryTranslation.modelfix": "Виправлення прозорих ліній в предметах, блоках\n§8✎ (без налаштувань)",
```
Comment (RozeFound) [inline Resourcepacks/Mod Menu Helper/assets/fo/lang/uk_ua.json:45]:
```suggestion
  "modmenu.summaryTranslation.modelfix": "Виправлення прозорих ліній в предметах, блоках\n§8✎ (лише продвинуті налаштування)",
```

**Comment by Den4enko** [comment]
Omg, I that Inattentive

**Review by RozeFound — APPROVED** [review]

**Pull request merged**